deepchem/feat/rdkit_grid_featurizer.py

```diff
@@ -620,8 +624,8 @@ def convert_atom_pair_to_voxel(molecule_xyz_tuple, atom_index_pair, box_width,
 def compute_charge_dictionary(molecule):
   """Computes partial charges for each atom."""
```

Comment:
Can you add to the docstring that this function assumes one has already called rdkit_util.compute_charges(mol).

Reply:
Oh, you're right, I missed it. I'll add a docstring better describing what the function actually does.